locales/en/email-strings.ftl

```diff
@@ -97,3 +97,26 @@ pre-fxa-message = <a>Create a free {-brand-fxa}</a>, and you can add up to 15 em
 
 # Section headline
 monitor-another-email = Want to monitor another email?
+
+
+# Subject line of email
+pre-fxa-subject = An update from {-product-name}
+
+pre-fxa-headline = What’s changing with {-product-name}
+pre-fxa-blurb = Here’s what’s changed since you signed up for {-product-name}, the service that 
```

Comment:
Sorry, I just realized we're doing `{-brand-name} accounts` for `Firefox accounts`. That's not great, because "Firefox account" as a whole should be a term.

Let's just use `Firefox accounts`. For the singular form, if needed, we should create a term.